src/main/java/org/owasp/html/ExtensibleHtmlStreamRenderer.java

```diff
@@ -0,0 +1,472 @@
+// Copyright (c) 2011, Mike Samuel
```

Comment:
Should license part also be added in new file...Please check any earlier file

Reply:
This is not a Zimbra class, I think we should maintain the original license text.

Reply:
left license as it is